packaging/windows/BUILD.txt

```diff
@@ -6,44 +6,61 @@ A) Native compile using MSYS2:  How to make a darktable windows installer (64 bi
 
     Install MSYS2 (instructions and prerequisites can be found on official website: https://msys2.github.io/).
     Update the base MSYS2 system until no further updates are available using:
-    $ pacman -Syu
+		$ pacman -Syu
 
-    Install x64 developer tools and x86_64 toolchain and git from the MSYS2 prompt:
-    $ pacman -S base-devel
-    $ pacman -S mingw-w64-x86_64-{toolchain,cmake,nsis}
-    $ pacman -S git
+    From MSYS2 terminal, install x64 developer tools and x86_64 toolchain and git from the MSYS2 prompt:
+		$ pacman -S base-devel
+		$ pacman -S mingw-w64-x86_64-{toolchain,cmake,nsis}
+		$ pacman -S git
 
     Install required libraries and dependencies:
-    $ pacman -S mingw-w64-x86_64-{exiv2,lcms2,lensfun,dbus-glib,openexr,sqlite3,libxslt,libsoup,libwebp,libsecret,lua,graphicsmagick,openjpeg2,gtk3,pugixml,libexif,osm-gps-map,libgphoto2,flickcurl,drmingw,gettext,python3,iso-codes}
+		$ pacman -S mingw-w64-x86_64-{exiv2,lcms2,lensfun,dbus-glib,openexr,sqlite3,libxslt,libsoup,libwebp,libsecret,lua,graphicsmagick,openjpeg2,gtk3,pugixml,libexif,osm-gps-map,libgphoto2,flickcurl,drmingw,gettext,python3,iso-codes,python3-jsonschema,python2-setuptools,python3-setuptools}
 
     For gphoto2:
     You need to restart the MINGW64 or MSYS terminal to have CAMLIBS and IOLIBS environment variables  properly set for LIBGPHOTO are available. You can check them with:
-    $ echo $CAMLIBS
-    $ echo $IOLIBS
-
-    Update your lensfun database:
-    $ lensfun-update-data
+		$ echo $CAMLIBS
+		$ echo $IOLIBS
+
+    From MINGW64 terminal, update your lensfun database:
```

Comment:
This is not needed here. It's enough to update the lensfun database **after** you download lensfun manually and fix the folder as you describe a few lines below.

Reply:
Correct, but sooner or later the lensfun issue will be fixed and there will be no need to downgrade.
By then we will remove the downgrade part and everything will be fine